logger.go

```diff
@@ -0,0 +1,10 @@
+package checkup
+
+import (
+	"io/ioutil"
+	"log"
+)
+
+func init() {
+	log.SetOutput(ioutil.Discard)
```

Comment:
We can, but it will change the behaviour. Presently, logging is always turned off unless the client code explicitly enables it. If we move this to the `root.go` file, then logging is always turned on unless the client code specifically turns it off. I figured users would find output from this package undesirable unless they explicitly turn it on. What do you think?